goals/law-practice-office-action-extraction-rung/SPEC.md

```
@@ -0,0 +1,122 @@
+      handling are either implemented after service extraction is green or
+      recorded as the next packet phase with no overclaim.
+- [ ] No real client fixtures or provider secrets are introduced.
+- [ ] No unrelated refactors or formatting churn.
+
+## Verification Matrix
+
+| Check | Command or evidence | Required result |
+| --- | --- | --- |
+| Packet launcher size | `test "$(wc -m < goals/law-practice-office-action-extraction-rung/GOAL.md)" -le 4000` | Passes |
+| Manifest JSON | `jq . goals/law-practice-office-action-extraction-rung/ops/manifest.json` | Passes |
+| Packet references | `rg -n "law-practice-office-action-extraction-rung|GOAL.md|agentLaunchers|packetAnchorDocument" goals/law-practice-office-action-extraction-rung` | Finds expected references |
```

markdownlint-cli2 (0.22.1)

[warning] Table column count
Expected: 3; Actual: 6; Too many cells, extra data will be missing

(MD056, table-column-count)